apps/src/templates/PaneHeader.jsx

```diff
@@ -143,6 +148,7 @@ export const PaneButton = Radium(function(props) {
     ...(props.isMinecraft && styles.headerButtonMinecraft),
     ...(props.isPressed && styles.headerButtonPressed),
     ...(!props.headerHasFocus && styles.headerButtonUnfocused),
+    ...(props.disabled && styles.disabled),
```

Comment:
Why is the whole PaneHeader disabled (as opposed to just a button)?  (Sorry, I think I must have the wrong mental model for how the components fit together.)